query-engine/request-handlers/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 
 [dependencies]
 prisma-models = { path = "../prisma-models" }
+nodejs-drivers = { path = "../nodejs-drivers" }
```

Comment:
this is missing an `optional = true`

Reply:
Unfortunately, this is not optional as of the implementation. Rather than conditionally compiled, `load_executor::load` depends on an optional parameter that is `NodeJSQueryable`. We could conditionally compile those, but it will lead to more complexity in the code, I believe. 

I'm seeing how to overcome this problem and make the `request_handlers` crate not to always depend on `node-js-drivers`